### DISCUSSION TITLE
GPU is not being used

### DISCUSSION THREAD

**Discussion by Aurenlian** [azuritecoin/OnnxDiffusersUI#14] [Q&A] [answered]
I managed to download and use the model and UI with no problem, but for some reason even though i didn't use --cpu-only, it still only uses CPU and normal RAM and does not use my RX 570, what is weird is when i downloaded this UI when it first came out it worked fine. Am i missing something or is that a bug?

**Comment by azuritecoin** [comment, marked as answer]
Hey, yeah it's a bug that I found yesterday. There's a simple fix. Go activate the virtual environment, then update the onnxruntime-directml package.  
```shell
.\virtualenv\Scripts\activate.bat
Pip install -I onnxruntime-directml
```
